### ISSUE TITLE
[BUG] Does not play well with DnD5e 2.0.3

### ISSUE THREAD

**Issue by ctbritt** [fantasycalendar/FoundryVTT-RestRecovery#92]
**Describe the bug**
no longer restores hit points, features, charges, etc.

**To Reproduce**
1. Enable Rest Recovery 5e
2. Click on Long rest icon
3. Gnash teeth

**Expected behavior**
Hit points and other items to be restored

**Screenshots**

**Setup:**
 - Foundry Version: 10.286
 - System Version: DND5e 2.0.3
 - Rest Recovery Version: 1.3.1
 - Browser & version Google Chrome

**Active modules:**
 - Rest Recovery
 - libwrapper

**Additional context**
Here's the error I get back: 
```
Uncaught (in promise) TypeError: this._getRestItemUsesRecovery is not a function or its return value is not iterable
[Detected 2 packages: rest-recovery, lib-wrapper]
    at 🎁CONFIG.Actor.documentClass.prototype._rest#rest-recovery (module.js:21246:19)
    at 🎁CONFIG.Actor.documentClass.prototype._rest#0 (libWrapper-wrapper.js:187:52)
    at 🎁CONFIG.Actor.documentClass.prototype.longRest#rest-recovery (module.js:21214:19)
```

**Comment by Haxxer** [comment]
Could you export the actor that causes this error and upload it please?

**Comment by ctbritt** [comment]
[fvtt-Actor-caliann-(of-house-chartyx).json.zip](https://github.com/fantasycalendar/FoundryVTT-RestRecovery/files/9629093/fvtt-Actor-caliann-.of-house-chartyx.json.zip)

there you go.

**Comment by Orionox** [comment]
I'm having this same issue on all my actors

**Comment by Haxxer** [comment]
Fixed in 1.3.2
